charts/airflow/README.md

```diff
@@ -1556,6 +1556,33 @@ Parameter | Description | Default
 <hr>
 </details>
 
+### `triggerer.*`
+<details>
+<summary>Expand</summary>
+<hr>
+
+Parameter | Description                                                                             | Default
+--- |-----------------------------------------------------------------------------------------| ---
+`triggerer.enabled` | if the triggerer should be deployed. `airflow.legacyCommands` should be `false` as well | `false`
```

Comment:
The default value is now `true`, and description is always the same as the first line of the docstring, so in this case:
```md
`triggerer.enabled` | if the triggerer should be deployed | `true`
```